### PR TITLE
Plugins: Add support for ResourceConversion

### PR DESCRIPTION
**Pull request by toddtreece**
**What is this feature?**

Update to add support for conversion introduced in https://github.com/grafana/grafana-plugin-sdk-go/pull/1044

**Why do we need this feature?**

A new `ResourceConversion` service was added to the plugin SDK.

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Related PR: https://github.com/grafana/grafana-enterprise/pull/7066

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

**Review by ryantxu — APPROVED** [review]

**Review by marefr — APPROVED** [review]
LGTM

File: pkg/plugins/backendplugin/coreplugin/core_plugin.go [hunk 1]
```diff
@@ -143,10 +144,10 @@ func (cp *corePlugin) ValidateAdmission(ctx context.Context, req *backend.Admiss
 	return nil, plugins.ErrMethodNotImplemented
 }
 
-func (cp *corePlugin) ConvertObject(ctx context.Context, req *backend.ConversionRequest) (*backend.ConversionResponse, error) {
+func (cp *corePlugin) ConvertObjects(ctx context.Context, req *backend.ConversionRequest) (*backend.ConversionResponse, error) {
 	if cp.AdmissionHandler != nil {
```
Comment (andresmgot) [inline pkg/plugins/backendplugin/coreplugin/core_plugin.go:148]: this check is now different, it should be `if cp.ConversionHandler != nil`

**Review by andresmgot — COMMENTED** [review]
I spotted a small thing 👍

File: pkg/plugins/backendplugin/coreplugin/core_plugin.go (hunk 1 above)
Comment (toddtreece) [reply pkg/plugins/backendplugin/coreplugin/core_plugin.go:148]: thanks! it looks like i missed a couple. updated in https://github.com/grafana/grafana/pull/92011

**Comment by briangann** [comment]
@toddtreece should we backport this (and any related PRs) to 11.2.x, trying to get that branch up to the current sdk would require it.

**Pull request merged**